youtube_dl/extractor/bandcamp.py

```diff
@@ -415,3 +408,52 @@ def _real_extract(self, url):
             'episode_id': compat_str(video_id),
             'formats': formats
         }
+
+
+class BandcampUserIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:user'
+    _VALID_URL = r'https?://(?:(?P<subdomain>[^.]+)\.)?bandcamp\.com/?$'
+
+    _TESTS = [{
+        'url': 'https://adrianvonziegler.bandcamp.com',
+        'info_dict': {
+            'id': 'adrianvonziegler',
+            'title': 'Albums of adrianvonziegler',
+        },
+        'playlist_mincount': 20,
+    }, {
+        'url': 'http://dotscale.bandcamp.com',
+        'info_dict': {
+            'id': 'dotscale',
+            'title': 'Albums of dotscale',
+        },
+        'playlist_count': 1,
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        uploader = mobj.group('subdomain')
+
+        webpage = self._download_webpage(url, uploader)
+
+        album_elements = re.findall(r'<a href="/album/(.+)">', webpage)
+
+        if not album_elements:
```

Comment:
Remove.

Reply:
@dstftw  Fixed in 31272f0724e363253d9d1fb96f25b0bd3f94606e

Thank you for your suggestions :)